src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Publish.targets

```diff
@@ -36,9 +36,26 @@ Copyright (c) .NET Foundation. All rights reserved.
     The main publish entry point.
     ============================================================
     -->
+
+  <PropertyGroup Condition="'$(NoBuild)' != 'true'">
```

Comment:
Should we use a less generic name for this property?  Perhaps `PublishWithoutBuild` or something?

Reply:
This matches pack --no-build so I think it's worth the consistency. Otherwise it should have been `PackWithoutBuilding`.